etc/org.opencastproject.ui.metadata.CatalogUIAdapterFactory-episode-common.cfg

```diff
@@ -144,3 +144,11 @@ property.license.readOnly=false
 property.license.required=false
 property.license.listprovider=LICENSES
 property.license.order=5
+
+# Created By
+property.createdBy.inputID=publisher
+property.createdBy.label=EVENTS.EVENTS.DETAILS.METADATA.CREATED_BY
+property.createdBy.type=text
```

Comment:
Since this pull focuses on "publisher" as the person initiating the publication of the event, it would avoid confusion with "creator" if the METADATA label and text was PUBLISHER (vs CREATED_BY). i.e avoid duplicating the craziness of the "#Presenters" section in this file.